### ISSUE TITLE
JC69 model name is wrong

### ISSUE THREAD

**Issue by GavinHuttley** [cogent3/cogent3#941]
indicates it's F81

```python
from cogent3 import get_model

sm = get_model("JC69")
sm.name
```
returns F81!

**Comment by GavinHuttley** [comment]
Note, this affects `K80` too

**Comment by jamesmartini** [comment]
Models JC69 and K80 are nested based on F81 and HKY85, respectively. Suggested fix made in pull request: change API so that the name argument is now in kwargs variable for the nested models, and make sure base models retain their name.

**Comment by GavinHuttley** [comment]
fixed in d87ccbd54ddca5bc7bbd5feddd064e4d01c7f597
